.circleci/config.yml

```diff
@@ -59,6 +59,9 @@ jobs:
       tag: 16.13-browsers
     steps:
       - checkout
+      - run:
+          name: Update NPM to match package.json
+          command: sudo npm install -g npm@$(jq -r '.engines.npm' < package.json)
```

Comment:
what's the effect of updating NPM then running npm install using a package-lock.json that was generated using the older version of NPM?

Reply:
The `package-lock.json` will always now be generated using the listed version.  If you update npm locally, you will have to run `npm i` to ensure it gets updated to whatever the appropriate format is.

Reply:
sorry, misread your question... basically you'll now need to update the version of npm in the `package.json` before running `npm i` when doing an update - it'll then update the lock file.